bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -91,6 +92,19 @@ impl LockTime {
         }
     }
 
+    /// Constructs a `LockTime` from the sequence number of a Bitcoin input.
+    ///
+    /// This method will **not** round-trip with [`Self::to_sequence`]. See the
+    /// docs for [`Self::from_consensus`] for more information.
+    #[inline]
+    pub fn from_sequence(n: Sequence) -> Result<Self, DisabledLockTimeError> {
+        Self::from_consensus(n.to_consensus_u32())
+    }
+
+    /// Encodes the locktime as a sequence number.
+    #[inline]
+    pub fn to_sequence(&self) -> Sequence { Sequence::from_consensus(self.to_consensus_u32()) }
```

Comment:
FTR these methods will force us to move `Sequence` to `units` too. I think it's fine.

Reply:
Hmm, good to be aware of. I think it's fine as well. `Sequence` isn't quite a unit, but its most common use is as a locktime, which is a unit, so ok.

Reply:
Not actionable, clicking "resolve".